### ISSUE TITLE
Sound in 1.2.25 and 1.2.25L

### ISSUE THREAD

**Issue by EngelMT** [FrSky-OS/HORUS-X10-X10S-X12S-bug-report#102]
Customers inform us about problems with sounds. It works without problems some minutes. But then it stops and only the Haptic Modul works. After landing the transmitter must switch out. After switch on it works.

Last information from a customer is that he makes 6 flights last weekend. All flights are without telemetrie sound and tracking sound after some minutes.

**Comment by EngelMT** [comment]
More exactly info to this issue. Customers tested now with 1.2.25 and after several flights with 1.2.25L. With 1.2.25 the telemetry contact lost one time in 10 flights. With 1.2.25L it lost 10 times in 8 flights.

So it is a problem of both software versions. But more critical with 1.2.25L.

**Comment by Adelaxie** [comment]
@EngelMT  telemetry lost (all telemetry data not updated) or just sounds missing??

**Comment by EngelMT** [comment]
Only sounds. Telemetrie ist well shown on display. And the haptic works also well. But sounds are lost and it is only possible to remove if the transmitter is switched out and on.

**Comment by Adelaxie** [comment]
ok，thanks

**Comment by Adelaxie** [comment]
OUT OF DATE ISSUES.

**Comment by EngelMT** [comment]
Sorry, but problem is not solved. Some customers have this problem up to now.

**Comment by Adelaxie** [comment]
really? I did not receive any of this kind , ok , reopen it and can you provide more information based on latest firmware please? thanks in advance.

**Comment by EngelMT** [comment]
Yes really. Some days in every fligths and on other day only one time. Or never in some weeks and than again sometimes on one day. So it is not easy to find out the problem.

**Comment by janekx** [comment]
SD card issue?

**Comment by EngelMT** [comment]
Tested with severl new and different SD-cards.
